docs/content/getting-started/quick-start.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Traefik Getting Started Quickly"
-description: "Looking to get started with Traefik Proxy quickly? Read the technical documentation to learn a simple use case that leverages Docker."
+description: "Get started with Traefik Proxy and Docker."
```

Comment:
See my above comment.

Reply:
Also, this is SEO and a description for documentation and should IMHO focus in the description on providing value for the user 😄.
Yes SEO is important but also UX and value for the user 😄.